extensions/amp-access-laterpay/0.1/test/validator-amp-access-laterpay.html

```diff
@@ -0,0 +1,71 @@
+<!--
```

Comment:
I added couple things, such as `amp-access-laterpay` script and vendor config. There's not much in terms of requirements here. Also, I imagine we may add a couple more things. One thing, for instance, we could add `id=amp-access-laterpay-dialog` requirement. Do you think it'd be good to add it in validator?

On the other hand, `laterpay` vendor config might be hard to enforce since we don't have JSON rules, do we?